tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -27,6 +27,17 @@ public function set_up() {
 		$this->interactivity = new WP_Interactivity_API();
 	}
 
+	/**
+	 * Tear down.
+	 */
+	public function tear_down() {
+		remove_filter( 'pre_option_blog_charset', array( $this, 'charset_iso_8859_1' ) );
+	}
+
```

Comment:
This shouldn't be necessary because filters are reset with each test already.

See in `set_up`: https://github.com/WordPress/wordpress-develop/blob/8d5e55ad1b8d4ba6f2cb1b3f71a18963aa5c84b4/tests/phpunit/includes/abstract-testcase.php#L111-L113

And in `tear_down`: https://github.com/WordPress/wordpress-develop/blob/8d5e55ad1b8d4ba6f2cb1b3f71a18963aa5c84b4/tests/phpunit/includes/abstract-testcase.php#L202

Also, importantly, this is missing `parent::tear_down()` if it were to be included.

```suggestion
```